### PR TITLE
add metadata on ipfs

### PR DESCRIPTION
**Pull request by Zett98**
### Description
store metadata on ipfs

### CODE REVIEW

File: context/metadata_blob.ts [hunk 1]
```diff
@@ -22,5 +22,26 @@ let exported = {
...
+  metadata: MichelsonMap<any, unknown>;
+}> {
+  const formData = new FormData();
+  let str = JSON.stringify(meta);
+  const bytes = new TextEncoder().encode(str);
+  const blob = new Blob([bytes], {
+    type: "application/json;charset=utf-8",
+  });
+  formData.append("file", blob, "");
+  let { cid } = await fetch("https://ipfs-proxy.gcp.marigold.dev/add", {
```
Comment (rueshyna) [inline context/metadata_blob.ts:35]: put url in config?

File: context/state.ts [hunk 2]
```diff
@@ -26,15 +32,20 @@ type storage = {
 
 let emptyState = () => {
   let connection = new TezosToolkit(RPC);
+  const customHandler = new Map<string, Handler>([
+    ["ipfs", new IpfsHttpHandler("ipfs.gcp.marigold.dev")],
```
Comment (rueshyna) [inline context/state.ts:36]: the same, put url in config?

**Review by rueshyna — COMMENTED** [review]
The rest looks good to me

File: context/metadata_blob.ts (hunk 1 above)
Comment (Zett98) [reply context/metadata_blob.ts]: done

File: context/state.ts (hunk 2 above)
Comment (Zett98) [reply context/state.ts]: done

**Pull request merged**